api-catalog-services/src/test/java/org/zowe/apiml/apicatalog/functional/ApiCatalogFunctionalTest.java

```diff
@@ -33,6 +37,10 @@ void setUp() {
     }
 
     protected String getCatalogUriWithPath(String path) {
-        return String.format("https://%s:%d", hostname, port) + path;
+        return getCatalogUriWithPath("https", path);
+    }
+
+    protected String getCatalogUriWithPath(String scheme, String path) {
+        return String.format("%s://%s:%d", scheme, hostname, port) + path;
```

Comment:
is it possible to use path as another parameter in format()?

Reply:
done